### ISSUE TITLE
Cannot resolve any validators with default configuration (Fluentvalidation)

### ISSUE THREAD

**Issue by hamaronooo** [FastEndpoints/FastEndpoints#751]
Hello!

_Specs:_
- FastEndpoints 5.28.0
- FlentValidation: 11.9.2

Have simple validator:
```
public class CreateGroupRequest
{
	public string Name { get; set; }
	public string Code { get; set; }
	public IEnumerable<GroupMemberEditCreateDto>? Members { get; set; }
}

public class CreateGroupRequestValidator : Validator<CreateGroupRequest>
{
	public CreateGroupRequestValidator()
	{
		RuleFor(x => x.Name).NotEmpty().NotNull().MaximumLength(128);
		RuleFor(x => x.Code).NotEmpty().NotNull().MaximumLength(100);
	}
}
```

and an Endpoint:
```
public sealed class Endpoint : Endpoint<CreateGroupRequest, Guid>
{
	public IMapper Mapper { get; set; }
	public IMediator Mediator { get; set; }
	public IOptions<AuthorizationSettings> AuthSettings { get; set; }

	public override void Configure()
	{
		Post("group");
		Version(1);
		Validator<CreateGroupRequestValidator>();
	}

	public override async Task<Guid> ExecuteAsync(CreateGroupRequest request, CancellationToken ct)
	{
		ThrowIfAnyErrors();
	        // some logic
	}
}
```

It's throwing Exception when Endpoint was called.
It can't find validator type because, by default, it's `scoped lifetime`:
```
Unhandled exception
System.InvalidOperationException: Cannot resolve scoped service 'WebApi.Endpoints.CreateGroupRequestValidator' from root provider.
   at Microsoft.Extensions.DependencyInjection.ServiceLookup.CallSiteValidator.ValidateResolution(ServiceCallSite callSite, IServiceScope scope, IServiceScope rootScope)
   at Microsoft.Extensions.DependencyInjection.ServiceProvider.GetService(ServiceIdentifier serviceIdentifier, ServiceProviderEngineScope serviceProviderEngineScope)
   at Microsoft.Extensions.DependencyInjection.ServiceLookup.ServiceProviderEngineScope.GetService(Type serviceType)
   at Microsoft.Extensions.DependencyInjection.ActivatorUtilities.GetServiceOrCreateInstance(IServiceProvider provider, Type type)
   at FastEndpoints.ServiceResolver.CreateSingleton(Type type)
   at FastEndpoints.EndpointDefinition.GetValidator()
   at FastEndpoints.Endpoint`2.ValidateRequest(TRequest req, EndpointDefinition def, List`1 validationFailures, CancellationToken cancellation)
   at FastEndpoints.Endpoint`2.ExecAsync(CancellationToken ct)
   at FastEndpoints.Endpoint`2.ExecAsync(CancellationToken ct)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|7_0(Endpoint endpoint, Task requestTask, ILogger logger)
```

⚠️ Solution for this 🪲 is explicitly set validators lifetime:
```
services.AddValidatorsFromAssemblyContaining<Program>(ServiceLifetime.Singleton);
```

Thanks!

**Comment by dj-nitehawk** [comment]
i'm unable to reproduce your issue.
works as expected:
![image](https://github.com/user-attachments/assets/65956bf7-4a87-4aa8-9b48-4c99ea59daa6)
can you update the following project to reproduce the issue and send back pls?
[Repro.zip](https://github.com/user-attachments/files/16530975/Repro.zip)

**Comment by hamaronooo** [comment]
Ohh... I researched your example, and it's true, there isn't same bug... 
I have a lot of additional packages in project, so I tried to add then one by one in your example.
After few packages I failed to reproduce in your example.
So, I think it's like a some side effect from someone of them...
I think it's closed. My mistake.

Here is screenshot of my packages list:
![image](https://github.com/user-attachments/assets/ce6e0d33-8722-4c52-965d-6c2321e8b215)

Thanks!
